tests/utils.js

```diff
@@ -0,0 +1,7 @@
+function get(selector, attribute = 'data-test') {
```

Comment:
This feels like far too generic a name. Kent Dodds uses `sel` but i feel like that's not very clear either

How about `byId`? Reads nicely i think:

```js
browser.click(byId("bar"));
cy.get(byId("foo"));
```

Or just simply `testId`

Reply:
In guess `byId` could be implemented in terms of `get`?

```javascript
function byId(selector) {
  return get(selector, 'id')
}
```

Reply:
I don't mean by actual `#id`, i mean test id. Just wanted a short name